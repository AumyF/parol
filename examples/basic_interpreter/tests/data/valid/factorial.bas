10 N = 6
20 RESULT = N
30 N = N - 1
40 IF N = 0 GOTO 90
50 RE = RE * N
60 GOTO 30
100 ? RESULT