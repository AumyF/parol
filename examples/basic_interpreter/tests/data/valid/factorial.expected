720 
N: 0, RE: 720